themes/default/files/scripts/display-base.js

```diff
@@ -273,6 +273,11 @@ Numbas.queueScript('display-base',['controls','math','xml','util','timing','jme'
                 }
             }),
             message: ko.computed(function() {
+
+                if (settings.showFeedbackIcon) {
```

Comment:
This is in the wrong place, and does the opposite of what it should!
You want to return `invisible` when `settings.showFeedbackIcon` is `false`, i.e. `if(!settings.showFeedbackIcon)`.

This test should be at the top of the `iconClass` observable, not here in `message`.